### PR TITLE
build: Update to header 1.3.297

### PR DESCRIPTION
**Pull request by mikes-lunarg**

### CODE REVIEW

File: layers/VkLayer_khronos_validation.json.in [hunk 1]
```diff
@@ -68,164 +68,559 @@
                 {
                     "label": "Standard",
                     "description": "Good default validation setup that balance validation coverage and performance.",
-                    "platforms": [ "WINDOWS", "LINUX", "MACOS", "ANDROID" ],
```
Comment (spencer-lunarg) [inline layers/VkLayer_khronos_validation.json.in:71]: so I made this change https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/8645

The auto formatting of this file is impossible to do real editing on. What is auto-formatting it for you, I would love to turn it off, the only thing that needs to be updated is the `api_version`

Comment (mikes-lunarg) [reply layers/VkLayer_khronos_validation.json.in:71]: That's a bummer. It's a python scrip that reads the json into a dict, modifies it, then dumps it back out to a json. I will revert the non-version changes and come up with a smarter way to do it automatically

Comment (spencer-lunarg) [reply layers/VkLayer_khronos_validation.json.in]:
```python
with open(file_path, 'r') as file:
    lines = file.readlines()
    with open(file_path, 'w') as file:
        for line in lines:
            if 'api_version' in line:
                # Modify the line with the new version
                line = f'        "api_version": "${new_version}"'
            file.write(line)
```

**Review by spencer-lunarg — APPROVED** [review]

File: layers/VkLayer_khronos_validation.json.in (hunk 1 above)
Comment (mikes-lunarg) [reply layers/VkLayer_khronos_validation.json.in]: https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/8667

**Pull request merged**